pkg/apis/eventing/v1beta1/trigger_types.go

```diff
@@ -92,7 +92,7 @@ type TriggerSpec struct {
 
 	// Delivery contains the delivery spec for this specific trigger.
 	// +optional
-	Delivery *eventingduckv1beta1.DeliverySpec `json:"delivery,omitempty"`
+	Delivery *eventingduckv1.DeliverySpec `json:"delivery,omitempty"`
```

Comment:
To be consistent with v1beta1 BrokerSpec.Delivery, shouldn't this remain eventingduckv1beta1? https://github.com/knative/eventing/blob/master/pkg/apis/eventing/v1beta1/broker_types.go#L80

Reply:
Ah ok, then I'm fine with it, thanks for the clarification